src/hackney_pool.erl

```diff
@@ -475,7 +475,7 @@ add_to_queue({_Host, _Port, _Transport} = Dest, From, Ref, Queues) ->
 del_from_queue({_Host, _Port, _Transport} = Dest, Ref, Queues) ->
   case dict:find(Dest, Queues) of
     error ->
-      {Queues, 0};
+      {Queues, false};
```

Comment:
What is it about?

Reply:
https://github.com/benoitc/hackney/pull/453, if there is the chance this PR gets close to being merged, it can be dropped (or rather merging https://github.com/benoitc/hackney/pull/453 is needed anyway so this solves its-self)